ts/indicators/acceleration-bands.src.ts

```diff
@@ -16,40 +16,40 @@ import H from '../parts/Globals.js';
  */
 declare global {
     namespace Highcharts {
-        interface AbandsIndicatorOptions extends SmaIndicatorOptions,
+        interface ABandsIndicatorOptions extends SMAIndicatorOptions,
             MultipleLinesIndicatorOptions {
-            params?: AbandsIndicatorParamsOptions;
+            params?: ABandsIndicatorParamsOptions;
             lineWidth?: number;
             topLine?: Dictionary<CSSObject>;
             bottomLine?: Dictionary<CSSObject>;
         }
 
-        interface AbandsIndicatorParamsOptions
-            extends SmaIndicatorParamsOptions {
+        interface ABandsIndicatorParamsOptions
+            extends SMAIndicatorParamsOptions {
             factor?: number;
         }
 
-        class AbandsIndicator
-            extends SmaIndicator implements MultipleLinesIndicator {
-            public options: AbandsIndicatorOptions;
-            public data: Array<AbandsIndicatorPoint>;
+        class ABandsIndicator
+            extends SMAIndicator implements MultipleLinesIndicator {
+            public options: ABandsIndicatorOptions;
+            public data: Array<ABandsIndicatorPoint>;
             public linesApiNames: MultipleLinesMixin['linesApiNames'];
```

Comment:
Not related to naming, but this one is lacking `pointClass` and `points` properties.

Reply:
Yes, there are a few things that I have on my list. Also some missing inheritance in another indicator. Just fix it in this PR or create another?

Reply:
As long as it goes into a separate commit, this PR can be used.